### PR TITLE
Updated OpenAPI specifications

### PR DESCRIPTION
**Pull request by ping-localhost**
### Scope of changes

Tried to add some of the missing components to the OpenAPI specification. It is probably not perfect, but should be a decent starting point for someone who wants to continue this.

Calls like Prepare/SendPrepared transaction are for example still missing from this MR.

### Type of change

- [ ] bug fix
- [ ] new feature
- [x] documentation
- [ ] other (describe)

### Acceptance criteria

N.A.

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.

### CODE REVIEW

**Comment by ping-localhost** [comment]
Second PR would be https://github.com/ping-localhost/envoy/pull/1, but it does change a lot (used Codestral to decrease the amount of work I had to do), so not sure about it yet.

File: pkg/web/static/openapi.json [hunk 1]
```diff
@@ -1520,6 +1817,250 @@
...
+          "crypto_address": {
+            "type": "string"
+          }
+        }
+      },
+      "Transfer": {
+        "type": "object",
+        "properties": {
+          "amount": {
+            "type": "number"
```
Comment (daniellemaxwell) [inline pkg/web/static/openapi.json:1924]:
```suggestion
            "type": "number",
            "format": "double",
```

File: pkg/web/static/openapi.yaml [hunk 2]
```diff
@@ -965,6 +1173,174 @@ components:
...
+          type: string
+        country:
+          type: string
+        crypto_address:
+          type: string
+    Transfer:
+      type: object
+      properties:
+        amount:
+          type: number
```
Comment (daniellemaxwell) [inline pkg/web/static/openapi.yaml:1250]:
```suggestion
          type: number
          format: double
```

File: pkg/web/static/openapi.json [hunk 3]
```diff
@@ -1520,6 +1817,249 @@
...
           }
         }
+      },
+      "Rejection": {
+        "type": "object",
+        "properties": {
+          "code": {
+            "type": "string",
+            "enum": [
+              "REJECTED",
```
Comment (daniellemaxwell) [inline pkg/web/static/openapi.json:1827]: The enum values have been updated to correspond with the what's listed in the UI and may be viewed [here](https://github.com/trisacrypto/envoy/blob/main/pkg/web/templates/components/transaction_rejection.html#L16-L30). A link to the all of the possible error codes has been added in a description. Let me know if you have any questions.

File: pkg/web/static/openapi.json [hunk 4]
```diff
@@ -1921,8 +1920,7 @@
         "type": "object",
         "properties": {
           "amount": {
-            "type": "number",
-            "format": "double"
+            "type": "number"
```
Comment (ping-localhost) [inline pkg/web/static/openapi.json:1923]: Did you change it back by accident?

File: pkg/web/static/openapi.yaml [hunk 5]
```diff
@@ -965,6 +1173,174 @@ components:
         prev_page_token:
           type: string
           format: base64
+    Rejection:
+      type: object
+      properties:
+        code:
+          type: string
+          enum:
+            - REJECTED
```
Comment (daniellemaxwell) [inline pkg/web/static/openapi.yaml:1182]: Same as above.

File: pkg/web/static/openapi.json (hunk 4 above)
Comment (daniellemaxwell) [reply pkg/web/static/openapi.json:1923]: I thought I'd pulled the changes. I'll take another look so that we can get this approved.

**Review by daniellemaxwell — APPROVED** [review]
Thank you for helping us keep the documentation up to date! It has been greatly appreciated it.

**Comment by ping-localhost** [comment]
Sorry for the extra pushes. I've rebased and signed the commit.

**Pull request merged**